sdk/spring/pipeline/cosmos-integration-matrix-for-cosmos.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
why don't we still call it sdk/spring/pipeline/cosmos-integration-matrix.json?

Reply:
because spring use the old one to test multiple versions